alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportDocLevelMonitorFanOutAction.kt

```diff
@@ -664,6 +663,26 @@ class TransportDocLevelMonitorFanOutAction
         )
     }
 
+    private fun publishBatchFindings(
+        monitor: Monitor,
+        findings: List<Finding>
+    ) {
+        val publishBatchFindingsRequest = PublishBatchFindingsRequest(monitor.id, findings)
+        AlertingPluginInterface.publishBatchFindings(
+            client as NodeClient,
+            publishBatchFindingsRequest,
+            object : ActionListener<SubscribeFindingsResponse> {
+                override fun onResponse(response: SubscribeFindingsResponse) {
+                    log.info("findings published successfully")
+                }
+
+                override fun onFailure(e: Exception) {
+                    log.error("findings published failed due to {}", e.message)
```

Comment:
plz never log like this
we are gobbling up stack trace
log.error("findings published failed.", e)